### DISCUSSION TITLE
Machinaris not finding my plots

### DISCUSSION THREAD

**Discussion by olivierLA75** [guydavis/machinaris#226] [Q&A] [answered]
I don't know what I'm doing wrong but Machinaris doesn't see/
<img width="397" alt="Screen Shot 2021-08-16 at 8 20 17 PM" src="https://user-images.githubusercontent.com/83734065/129658609-23e9b093-674b-4cf0-8a5b-d32a9e2951ee.png">
<img width="398" alt="Screen Shot 2021-08-16 at 8 21 12 PM" src="https://user-images.githubusercontent.com/83734065/129658612-9cdec5bc-8a63-45b3-85cf-c345349d7f8d.png">
<img width="453" alt="Screen Shot 2021-08-16 at 8 21 50 PM" src="https://user-images.githubusercontent.com/83734065/129658617-44e19201-c997-4f5d-9653-e03addaf2011.png">
find my plots on Ubuntu

**Comment by olivierLA75** [comment]
and this one
<img width="825" alt="Screen Shot 2021-08-16 at 8 23 50 PM" src="https://user-images.githubusercontent.com/83734065/129658751-72b71c00-112b-4383-8771-2b7fa70f0d1a.png">

**Reply by guydavis** [reply]
Hi, what does `docker container inspect machinaris` show?

Also please paste output from:
```
ls -al /media/sm2/SM2Mining1
docker exec -it machinaris bash
ls -al /plots
```
I wonder if this is a permission issue? Thanks!

**Comment by olivierLA75** [comment]
<img width="933" alt="Screen Shot 2021-08-16 at 8 47 44 PM" src="https://user-images.githubusercontent.com/83734065/129660601-f9a304bb-1fb6-42a6-8a5d-94614b7552d4.png">
<img width="502" alt="Screen Shot 2021-08-16 at 8 46 41 PM" src="https://user-images.githubusercontent.com/83734065/129660616-a664a96a-daa6-41ea-bb03-3964b5ef18c8.png">
<img width="944" alt="Screen Shot 2021-08-16 at 8 47 55 PM" src="https://user-images.githubusercontent.com/83734065/129660619-30681fa9-c779-4f2f-b95d-5b04c9ae2135.png">

**Reply by guydavis** [reply]
Hi, it looks like you should enter this instead as you are one level up in your provided config:

-v /media/sm2/SM2Mining1/plots:/plots

**Reply by olivierLA75** [reply]
thanks! how do I do that? do I have to stop the container first and use the launch wizard again?

**Comment by guydavis** [comment]
Yes, just change the Docker run command and re-run after first stop and rm.

You provided a host volume path of "/media sm2/SM2Mining1" as folder
holding your plot files, but it looks like "/media sm2/SM2Mining1/plots"
actually holds them. Guessing based on your screenshots.

**Comment by olivierLA75** [comment]
I changed it but it still can't see them. could it be because the wallet/machinaris isn't synchronized yet?

**Reply by guydavis** [reply]
Hi, after you made the change I suggested, what does `docker container inspect machinaris` show?

Also please paste output from:
```
ls -al /media/sm2/SM2Mining1/plots
docker exec -it machinaris bash
ls -al /plots
```
Thanks!

**Comment by olivierLA75** [comment, marked as answer]
I think the previous container was still working. it seems to work now. thanks for checking!
I have another question though, I have a server that I'm going to use as a worker with 4 processors and 4 NVMes of 4tb each. it can process 40 plots at once and I was wondering what would be the best way to set it up with Machinaris, should I make one volume of 16tb with the NVMes or keep 4 different volumes and have temp, temp2, temp3 and temp4? not sure if it can handle 4 temps...

**Reply by guydavis** [reply]
You'll want to test your system to find the best settings.  Here are some [details on RAIDing SSDs](https://github.com/guydavis/machinaris/wiki/Hardware#raid-0-to-combine-ssds).
